CHANGES.md

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ----------
 
+- Remove reliance on shell for bootstrapping. Previously, we'd use the shell to
+  get the number of processors. (#7274, @rgrinberg)
+
```

Comment:
Maybe just merge both our entries.

Reply:
Why? One is a general improvement and the other is a bug fix. If they are merged, it would only obfuscate it for readers.